db/migrate/20230311033148_create_chapters.rb

```diff
@@ -0,0 +1,19 @@
+class CreateChapters < ActiveRecord::Migration[7.0]
+  def change
+    create_table :chapters do |t|
+      t.references :book, :foreign_key => true, :null => false
```

Comment:
I also suggest doing `, :index => true` on this column. 

It will help optimize the search because we are going to be looking up Chapters by `book_id` pretty often. 

* It's good because it makes it fast
* It's bad, because it actually makes the database heavier in size 
* Therefore, you shouldn't just add it on everything. Only the things that you know will have be used to query a lot. (You can always add it in later).